docs/lib/project-setup/fragments/native_common/prereq/flutter_null_safety.md

```diff
@@ -1,13 +1,6 @@
 
-<amplify-callout warning>
+<amplify-callout>
```

Comment:
Actually, I guess I should call out that the follow-up PR is just removing these callouts altogether except where we talk about codegen. Seeing as how null safety will become the new default, I think it's probably reasonable to not mention it explicitly going forward?

Reply:
I think we should leave these up, at least temporarily.

Reply:
I agree we should leave it in for now, especially while its not enabled by default in the CLI codegen for existing projects. Right now its opt-in for existing projects. Its only enabled by default for new projects.

Once we move to opt-out as the default for all projects, we can remove these callouts.

Reply:
Ok, I will restore the callouts and let this PR update the copy